styles.css

```diff
@@ -161,6 +160,296 @@
     align-items: center;
     justify-content: center;
   }
+
+  /* My Works */
+  .works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
- [ ] The main container of the projects should be styled with display: grid rather than flexbox. Kindly implement this.